packages/EasyCore/tests/Bridge/Symfony/Serializer/TrimStringsDenormalizerTest.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Tests\Bridge\Symfony\Serializer;
+
+use EonX\EasyCore\Bridge\Symfony\Serializer\TrimStringsDenormalizer;
+use EonX\EasyCore\Helpers\StringsTrimmerInterface;
+use EonX\EasyCore\Tests\Bridge\Symfony\AbstractSymfonyTestCase;
+use Mockery\MockInterface;
+use stdClass;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+/**
+ * @covers \EonX\EasyCore\Bridge\Symfony\Serializer\TrimStringsDenormalizer
+ *
+ * @internal
+ */
+final class TrimStringsDenormalizerTest extends AbstractSymfonyTestCase
+{
+    /**
+     * @return mixed[]
+     *
+     * @see testSupportsDenormalizationReturnsTrue
+     */
+    public function provideCorrectDataForDenormalization(): array
+    {
+        return [
+            'context has not `TRIM_STRINGS_ALREADY_CALLED` key' => [
```

Comment:
I don't think we need this set. You could just add non-empty context for the other two sets.

Reply:
Fixed.